dev-python/snakemake/snakemake-5.31.1.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Make-like task language"
+HOMEPAGE="https://snakemake.readthedocs.io"
+SRC_URI="https://github.com/${PN}/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# missing test dep google.cloud
+RESTRICT="test"
+
+BDEPEND="test? (
+	dev-python/requests-mock[${PYTHON_USEDEP}]
+	dev-python/google-api-python-client[${PYTHON_USEDEP}]
+	dev-python/jinja[${PYTHON_USEDEP}]
+	dev-python/networkx[${PYTHON_USEDEP}]
+	dev-python/pygments[${PYTHON_USEDEP}]
+	dev-python/pygraphviz[${PYTHON_USEDEP}]
```

Comment:
This feels very much like it is going to be doing network calls in the testing phase >.>

Reply:
Yes I think so too, which is why I'm not bothering with packaging the missing dependency, the testsuite will most likely fail anyway. I just added RESTRICT="test"